pythnet/message_buffer/idl/mock_cpi_caller.ts

```diff
@@ -0,0 +1,727 @@
+export type MockCpiCaller = {
```

Comment:
I don't think anyone is gonna use this idl

Reply:
ok i'll remove it ser.